src/charm.py

```diff
@@ -224,11 +230,13 @@ def _alerting_config(self):
             logger.debug('No alertmanagers available')
             return alerting_config
 
-        if len(self._stored.alertmanagers) > 1:
-            logger.warning('More than one altermanager found. Using first!')
+        targets = []
+        for manager in self.stored.alertmanagers:
+            port = self.stored.alertmanager_port
+            targets.append(f'{manager}:{port}')
```

Comment:
For Python 3.5 compatibility
```suggestion
            targets.append("{}:{}".format(manager, port))
```